### PR TITLE
lock swerves in auto/end of match

### PR DESCRIPTION
**Pull request by OliverW10**

### CODE REVIEW

File: controllers/movement.py [hunk 1]
```diff
@@ -189,6 +190,15 @@ def is_at_goal(self) -> bool:
     def manualdrive(self) -> None:
         if self.debug_trajectory:
             self.generate_trajectory()
+
+        # if we arent trying to move and almost at end of a match
+        if (
+            wpilib.DriverStation.getMatchTime() < 5
+            and sum(abs(x) for x in self.driver_inputs) < 0.01
```
Comment (auscompgeek) [inline controllers/movement.py:197]: nit: clearer to use map here
```suggestion
            and sum(map(abs, self.driver_inputs)) < 0.01
```

File: controllers/movement.py [hunk 2]
```diff
@@ -189,6 +190,15 @@ def is_at_goal(self) -> bool:
     def manualdrive(self) -> None:
         if self.debug_trajectory:
             self.generate_trajectory()
+
+        # if we arent trying to move and almost at end of a match
```
Comment (auscompgeek) [inline controllers/movement.py:194]: typo:
```suggestion
        # if we aren't trying to move and almost at end of a match
```

File: controllers/movement.py [hunk 3]
```diff
@@ -189,6 +190,15 @@ def is_at_goal(self) -> bool:
...
+        if (
+            wpilib.DriverStation.getMatchTime() < 5
```
Comment (auscompgeek) [inline controllers/movement.py:196]: This needs to account for when we're running the robot outside of a match.

Comment (OliverW10) [reply controllers/movement.py:196]: what is the best way to do this check?

Comment (auscompgeek) [reply controllers/movement.py:196]: Isn't match time documented to be some sort of sentinel value when not in a match or practice mode?

Comment (OliverW10) [reply controllers/movement.py:196]: If there is I couldn't find it

**Pull request merged**